Description: Set compile flags
 [Hardening] For the use of dpkg-buildflags all compile flags set by
 Debian must be added to configure.ac before creating all Makefiles.
Author: Joachim Wiedorn <joodebian at joonet.de>
Forwarded: no
Last-Update: 2016-03-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff -urN s06/configure.ac s07/configure.ac
--- s06/configure.ac	2012-05-13 10:28:46.000000000 +0200
+++ s07/configure.ac	2012-05-17 16:06:58.099533663 +0200
@@ -56,8 +56,9 @@
 AC_SUBST(LT_REVISION)
 AC_SUBST(LT_AGE)
 
-# Prevents setting flags.
-CXXFLAGS=""
+# Setting flags of Debian rules
+CXXFLAGS=${CXXFLAGS}
+LDFLAGS=${LDFLAGS}
 
 # More secret source
 AC_GNU_SOURCE
